src/Appwrite/Platform/Workers/Mails.php

```diff
@@ -63,6 +63,11 @@ public function action(Message $message, Registry $register, Log $log): void
         $name = $payload['name'];
         $body = $payload['body'];
 
+        $protocol = App::getEnv('_APP_OPTIONS_FORCE_HTTPS') == 'disabled' ? 'http' : 'https';
+        $hostname = App::getEnv('_APP_DOMAIN');
+
+        $body = str_replace(['{{protocol}}', '{{hostname}}'], [$protocol, $hostname], $body);
```

Comment:
Let's instead add it to `variables` array. Code below ensures all placehodlers are replaced with variables in both body, and subject.